Simplified content for self-hosted installation guides (#373)
okteto/docs: src/content/self-hosted/gke.mdx

- A Kubernetes cluster
- A working installation of kubectl
- A working installation of Helm v3
- An Okteto License (optional)

### Subdomain

> You'll need access to a [subdomain](self-hosted/administration/configuration.mdx#subdomain) to which you can add a wildcard DNS record, such as dev.example.com.

Reviewer: ```suggestion
You'll need sufficient access to a [subdomain](self-hosted/administration/configuration.mdx#subdomain) to add a wildcard DNS record, such as dev.example.com.
```

> By default, all endpoints created by Okteto for your development environments will be exposed in this wildcard subdomain.

Reviewer: ```suggestion
By default, all endpoints created by Okteto for your development environments will be exposed on the wildcard subdomain that you use.
```

> This guide will assume that your domain is registered in Google's [Cloud DNS](https://cloud.google.com/dns).

Reviewer: ```suggestion
This guide assumes your domain is registered in Google's [Cloud DNS](https://cloud.google.com/dns) service.
```

> Other DNS services can be used as well, but are not covered by this guide.

Reviewer: ```suggestion
You can use any DNS service you prefer, but this guide focuses specifically on Google's Cloud DNS.
```

We might even consider using the admonitions `info` callout here just to make sure this is obvious :)

### Getting your Okteto License

Okteto is free for up to 3 users without a license.

> You can also sign on for the [free trial](https://www.okteto.com/free-trial/) to give access up to 30 users for a month.

Reviewer: ```suggestion
You can also sign up for our [free trial](https://www.okteto.com/free-trial/), which supports up to 30 users for a month.
```

## Sign in to your Okteto instance

After a successful installation, you can access your Okteto instance at `https://okteto.SUBDOMAIN`. Your account will be automatically created as part of the login process. The first user to successfully login into the instance will be automatically assigned the `administrator` role.

> **Important**: The default installation is not recommended for production use. We highly advise following the steps outlined in the [Admin Dashboard](administration/dashboard.mdx) before giving your team access to your Okteto instance.

> The default installation is all you need to try our [getting started](getting-started.mdx) guide.

Author: @omnomagonz what do you think about the final CTA? and its tone?

Reviewer: ```suggestion
The default installation is all that's required to follow our [getting started](getting-started.mdx) guide.
```

Minor, non-blocking suggestion, but otherwise it sounds good to me 👍
